integration-cli/daemon/daemon.go

```diff
@@ -758,7 +758,7 @@ func (d *Daemon) ReloadConfig() error {
 }
 
 // WaitInspectWithArgs waits for the specified expression to be equals to the specified expected string in the given time.
-// FIXME(vdemeester) Attach this to the Daemon struct
+// Deprecated: use cli.WaitCmd instead
 func WaitInspectWithArgs(dockerBinary, name, expr, expected string, timeout time.Duration, arg ...string) error {
```

Comment:
yep it's is almost the same but it returns an error instead of using `testingT`. The amount of `wait…` in the code is big, didn't want to replace them all in that one PR 👼. That's why I put deprecated on it, and will definitely get removed soon (in follow-up PRs that will remove this func)